### ISSUE TITLE
Are data instances contexts and vice versa?

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/ontology#296]
We would like to clarify the relationship between data instances and context snippets. 

The question is posed in cell 4B of the Biography spreadsheet for Education: 
https://docs.google.com/spreadsheets/d/1L9V__vfiZ2hEZOsbVkMZ7-JlsToHHOuOKzbPIi-HTPs/edit#gid=1623510226

We see an inconsistency between our two diagrams, here:
https://drive.google.com/drive/folders/0B8C90DSaQG_sazJtMmVHc0s3M0k
and here:
https://drive.google.com/drive/folders/0B8C90DSaQG_sazJtMmVHc0s3M0k

The same thing seems to us to be called MEInstitutionalEducation and MEInstitutionalEducationContext

Are they the same thing or not?

**Comment by joelacummings** [comment]
The links for the drive are the same (link to a folder) but the description in the cell sounds right in that it would be an education context that is attached to the person.

**Comment by SusanBrown** [comment]
The images are:
https://drive.google.com/file/d/1q8Qu5imN4RB83a9EblLG4uNUdXg-Slnv/view?usp=sharing
I think this one is correct.

In the other:
https://drive.google.com/drive/folders/0B8C90DSaQG_sazJtMmVHc0s3M0k
I think that what is referred to here as MEInstitutionalEducation is synonymous with what we now call in the first diagram MEInstitutionalEducationContext, and "construct" here is a red herring. Please confirm.

**Comment by joelacummings** [comment]
Yes context is correct, construct is what I used before we solidified our terminology. The first link looks correct to me based on what we discussed.

**Comment by SusanBrown** [comment]
Great, thanks!

@lemaka I think we need to set up a subfolder of the Figures folder called "approved" in which we put the diagrams that reflect the current ontology. We'll need to revisit/revise some of them from time to time, but I think it would help to reduce some of the confusion around what is current. Could you please set that up, move in all figures you think are final (and put any you're not sure about on our agenda for tomorrow) and close this issue when you have done so?
